doc/classes/Node3D.xml

```diff
@@ -177,6 +177,13 @@
 				Rotates the local transformation around the Z axis by angle in radians.
 			</description>
 		</method>
+		<method name="scale_object">
+			<return type="void" />
+			<param index="0" name="ratio" type="Vector3" />
+			<description>
+				Scales the local transformation, relative to the parent's axes, by given 3D scale factors.
```

Comment:
Same point as raised in https://github.com/godotengine/godot/pull/90454 here, mentioning the parent is potentially confusing, I'd just keep the local part, it's clear enough and doesn't imply a parent is needed